src/entrypoints/config.ts

```diff
@@ -0,0 +1,29 @@
+export interface EntrypointConfig {
+    networkProviderUrl: string;
+    networkProviderKind: string;
+    chainId: string;
+}
+
+export class TestnetEntrypointConfig implements EntrypointConfig {
```

Comment:
I think it would be easier to have constructors on these classes in case somebody wants to change one of the fields (e.g. `networkProviderUrl`) without having to access the fields after the initialization.